spec/features/item_spec.rb

```diff
@@ -74,35 +74,53 @@
     expect(page.all("select#filters_in_category option").map(&:text).select(&:present?)).not_to eq(expected_order.reverse)
   end
 
-  describe "Item Table Tabs >" do
-    let(:item_name_1) { "the most wonderful magical pullups that truly potty train" }
-    let(:item_name_2) { "blackbeard's rugged tampons" }
-    before :each do
-      @item = create(:item, name: item_name_1, category: "same")
-      @item2 = create(:item, name: item_name_2, category: "different")
-      @storage = create(:storage_location, :with_items, item: @item, item_quantity: 666, name: "Test storage")
+  describe "Item Table Tabs >", focus: true do
+    let(:item_pullups) { create(:item, name: "the most wonderful magical pullups that truly potty train", category: "Magic Toddlers") }
+    let(:item_tampons) { create(:item, name: "blackbeard's rugged tampons", category: "Menstrual Products") }
+    let(:storage_name) { "the poop catcher warehouse" }
+    let(:storage) { create(:storage_location, :with_items, item: item_pullups, item_quantity: num_pullups_in_donation, name: storage_name) }
+    let!(:aux_storage) { create(:storage_location, :with_items, item: item_pullups, item_quantity: num_pullups_second_donation, name: "a secret secondary location") }
+    let(:num_pullups_in_donation) { 666 }
+    let(:num_pullups_second_donation) { 1 }
+    let(:num_tampons_in_donation) { 42 }
+    let(:num_tampons_second_donation) { 17 }
```

Comment:
Create different item totals in different StorageLocations, and verify the page is summing them up correctly.